documentation/modules/rn-2.2.adoc

```diff
@@ -39,3 +51,7 @@ Retaining the `Importer` pod for debug purposes causes warm migration to hang du
 As a temporary workaround, the `Importer` pod is removed at the end of the precopy stage so that the precopy succeeds. However, this means that the `Importer` pod log is not retained after warm migration is complete. You can only view the `Importer` pod log by using the `oc logs -f <cdi-importer_pod>` command during the precopy stage.
 
 This issue only affects the `Importer` pod log and warm migration. Cold migration and the `virt-v2v` logs are not affected.
+
+.Migration plan for VM with no OS fails with unclear error message
+
+In this release, when migration plans for VMs with no OS fail at the last step, the error message is `Convert image to kubevirt` without indicating the reason for the failure.
```

Comment:
If you are documenting a bug, you need to include the bug number.

No need for "In this release".

The description seems verbose. Please make it more concise.